### DISCUSSION TITLE
zh-cn file garbled

### DISCUSSION THREAD

**Discussion by opierice** [TownyAdvanced/Towny#7365] [Q&A] [answered]
When I opened the zh-cn.yml file, the translations inside were all garbled. I am using vscode to open the file encoded as gbk, gb18030, etc., but it still cannot be displayed properly. I want to know how to read this file

**Comment by LlmDl** [comment, marked as answer]
I use TextPad, but you can use any text editor that can read the file's actual encoding (UTF-8).
